tests/testthat/test-gather_images.R

```diff
@@ -0,0 +1,10 @@
+test_that("gather_images works", {
+  
+  species <- c("Mus_musculus","Pan_troglodytes","Homo_sapiens")
+  res <- rphylopic::gather_images(species=species)
```

Comment:
Please don't namespace calls to the package itself in tests (testthat namespacing makes sense though)

Reply:
is there a reason for this? i find it quite helpful to make it explicit where each function comes from, and to avoid namespace conflicts with other packages

Reply:
It's just my convention. no worries